MapboxNavigation/PollyVoiceController.swift

```diff
@@ -166,27 +172,29 @@ public class PollyVoiceController: RouteVoiceController {
         }
     }
     
-    func callSuperSpeak(_ text: String, error: String) {
+    func speakWithoutPolly(_ instruction: SpokenInstruction, error: Error) {
         pollyTask?.cancel()
         
+        voiceControllerDelegate?.voiceController?(self, spokenInstructionsDidFailWith: error)
+        
         guard let audioPlayer = audioPlayer else {
-            super.speak(fallbackText, error: error)
+            super.speak(instruction)
             return
         }
         
         guard !audioPlayer.isPlaying else { return }
         
-        super.speak(fallbackText, error: error)
+        super.speak(instruction)
     }
     
     func handle(_ awsTask: AWSTask<NSURL>) {
```

Comment:
Pass `instruction` into this method, so that it doesn’t need to use `lastSpokenInstruction`. This keeps the class minimally dependent on instance properties.